airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/DefaultJobPersistence.java

```diff
@@ -454,6 +454,22 @@ private static String current_timestamp() {
     return exportDatabase(DEFAULT_SCHEMA);
   }
 
+  @Override
+  public Map<String, Stream<JsonNode>> exportEverythingInDefaultSchema() throws IOException {
+    return exportEverything(DEFAULT_SCHEMA);
+  }
+
+  private Map<String, Stream<JsonNode>> exportEverything(final String schema) throws IOException {
```

Comment:
this seems very similar to `exportDatabase`. is this implementation different because we don't want to reference `DatabaseSchema.java` since that might be at a later version? if so, I would leave a comment explaining this.